File: day18/challenge_four.js
let objectOfStuff = {
  blue: "falcon",
  red: "devil",
  pink: "ranger",
  green: "lantern",
};

function objectPrinter(object) {
  var objectList = [];
  for (k in object) {
    if (object.hasOwnProperty(k)) {
      objectList.push(object[k]);
    }
  }
  return objectList;
}
console.log(objectPrinter(objectOfStuff));

File: day18/challenge_four.py
object_of_stuff = {
  "blue": "falcon",
  "red": "devil",
  "pink": "ranger",
  "green": "lantern",
}


def object_printer(obj):
    obj_list = []
    for k in obj:
        obj_list.append(obj[k])
    return obj_list


print(object_printer(object_of_stuff))
